likecoin/admin/post.php

```diff
@@ -178,7 +179,11 @@ function likecoin_get_post_tags( $post ) {
 	if ( ! $post_tags ) {
 		$post_tags = array();
 	}
-	return array_map( $func, $post_tags );
+	$result = array_map( $func, $post_tags );
+	if ( $limit ) {
+		$result = array_slice( $result, 0, 5 );
```

Comment:
Should this utilize `$limit`?
```suggestion
		$result = array_slice( $result, 0, $limit );
```